src/book.py

```diff
@@ -83,7 +83,7 @@ def _read_binance(self, file_path: Path) -> None:
                         "Could not recognize operation `%s` in binance file `%s:%i`.", operation, file_path, row)
                     continue
 
-                o = Op(utc_time, platform, change, coin)
+                o = Op(utc_time, platform, change, coin, reader.line_num)
```

Comment:
Oh nice, I didn't know about `line_num`. We should replace the already used `row` variable with this one. Then we no longer need enumerate.

We cold even start with `row = reader.line_num` to keep the variable short and handy and remove enumerate.